db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2025_05_23_031839) do
+ActiveRecord::Schema[7.1].define(version: 2025_05_29_031839) do
```

Comment:
@tds-1 I can't see the migration file for this change. Did you miss pushing this?